velox/functions/sparksql/String.h

```diff
@@ -1019,13 +1023,17 @@ struct ConvFunction {
     // When toBase is negative, converts to signed value. Otherwise, converts to
     // unsigned value. Overflow is allowed, consistent with Spark.
     if (toBase < 0) {
+      auto isMinInt64Num =
+          unsignedValue == (uint64_t)std::numeric_limits<int64_t>::min();
       int64_t signedValue;
-      if (isNegativeInput) {
+      if (isMinInt64Num) {
+        signedValue = (int64_t)unsignedValue;
```

Comment:
Seems it is of the same behavior as the else clause, maybe merge them into one. And can we rely on the `if (isNegativeInput)` clause to calculate the signedValue? Why do we need an extra branch?

Reply:
it is used to avoid unnecessary `std::abs`. if the input is isNegativeInput, then it can also be the `isMinInt64Num`.

Reply:
Maybe can be merged as below.
```
    if (isMinInt64Num || !isNegativeInput) {
       signedValue = (int64_t)unsignedValue;
    } else {
       signedValue = -std::abs((int64_t)unsignedValue);
    }
```

Reply:
addressed

Reply:
Thanks. I have no other comment.